vsock-proxy/enclave/src/app_configuration.rs

```diff
@@ -44,57 +44,58 @@ const CREDENTIALS_FILE: &str = "credentials.bin";
 
 const LOCATION_FILE: &str = "location.txt";
 
+const APPLICATION_CONFIGURATION_ID_LENGTH: usize = 32;
+
 pub(crate) fn setup_application_configuration<T>(
     em_app_credentials: &EmAppCredentials,
     ccm_backend_url: &CCMBackendUrl,
     api: T,
     fs_root: &Path,
-    app_config_id: Blob
+    app_config_id: &str,
```

Comment:
This is odd. A `Blob` (and the rest of the code) makes this assume the raw sha256 values. A `&str` makes me think this sha256 value is encoded in some way (e.g., base64).